src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/ApplicationInsightsTelemetryExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.FeatureManagement.FeatureFilters;
+
+namespace Microsoft.FeatureManagement.Telemetry.ApplicationInsights
+{
+    /// <summary>
+    /// Provides extension methods for tracking events with TargetingContext.
```

Comment:
```suggestion
    /// Provides extension methods for tracking events with <see cref="TargetingContext"/>.
```

Reply:
As mentioned in https://github.com/microsoft/FeatureManagement-Dotnet/pull/347#discussion_r1444962105
Is there any reason to use `TargetingContext" instead of "ITargetingContext" here?

Reply:
Contextual filters describe an interface that they handle. This allows a single app context type to satisfy multiple contextual feature filters.

Reply:
Don't take this as the actual answer, because I'm not sure what use case was in mind originally, but my understanding is that it enables something like:

```csharp
featureManager.IsEnabledAsync("Flag", new ContextRetriever());
```

```csharp
public class ContextRetriever: ITargetingContext
{
    string UserId { 
        get { ReadFromSomeLocalContext() } 
    }

    IEnumerable<string> Groups { 
        get { ReadFromSomeServer() }; 
    }
}
```

Which isn't needed in the Accessor Pattern, as the Accessor is essentially solving the same problem.